lib/portlayer/exec/handle.go

```diff
@@ -204,6 +204,17 @@ func (h *Handle) Commit(ctx context.Context, sess *session.Session, waitTime *in
 	return nil
 }
 
+// EmergencyCommit sets the handle's spec to nil so that Commit operation only does a state change and won't touch the extraconfig
+func (h *Handle) EmergencyCommit(ctx context.Context, sess *session.Session, waitTime *int32) error {
```

Comment:
Is there a need for this? Since there is no locking involved, you can just set `h.Spec = nil` before calling `Commit`. The name `EmergencyCommit` is kind of unusual as well.